Fix party disband on leave

marioded opened the pull request:
Now the party won't disband if the owner leaves an arena.

marioded committed:
Fix party disband on leave

Now the party won't disband if the owner leaves an arena.

andrei1058 commented:
You're breaking a lot of things by that. Like next-game party. Test it yourself. Deeper changes need to be made in order to keep the party session.

marioded committed:
Update Arena.java

marioded commented:
> You're breaking a lot of things by that. Like next-game party. Test it yourself. Deeper changes need to be made in order to keep the party session.

I can see only these 2 parts about the disband of the party, where's the rest?

Commit:
Update:
- Option to enable halloween features while in halloween period through the config.yml
- Added Unbreakable flag trough NMS to bows, swords and axes.
- Optimized a bit the main command
- Added the Play Again command (bw playagain, executable only in spectator, and I believe only through the spectator item) and the languages lines to English and Italian, other languages needs to be updated
- Fixed spaces on the plugin.yml LOL it was triggering me
- Added permissions to /bw join and /bw gui, now the permissions are: bw.join.GROUP or bw.join.random, and bw.gui.GROUP
- Fixed /bw arenagroup list
- Now the parties won't disband if the owner leaves an arena

ALL OF THESE THINGS WENT TESTED WITH 5 PLAYERS ONLINE - Thanks to Xx_ruben2005_xX for the support

Next updates by me: Hotbar manager, Map selector with favourites & challanges.

Commit:
Update:
- Option to enable halloween features while in halloween period through the config.yml
- Added Unbreakable flag trough NMS to bows, swords and axes.
- Optimized a bit the main command
- Added the Play Again command (bw playagain, executable only in spectator, and I believe only through the spectator item) and the languages lines to English and Italian, other languages needs to be updated
- Fixed spaces on the plugin.yml LOL it was triggering me
- Added permissions to /bw join and /bw gui, now the permissions are: bw.join.GROUP or bw.join.random, and bw.gui.GROUP
- Fixed /bw arenagroup list
- Now the parties won't disband if the owner leaves an arena

ALL OF THESE THINGS WENT TESTED WITH 5 PLAYERS ONLINE - Thanks to Xx_ruben2005_xX for the support

Next updates by me: Hotbar manager, Map selector with favourites & challanges.

Commit:
Merge remote-tracking branch 'BedWars1058/master'

Commit:
Update:
- Option to enable halloween features while in halloween period through the config.yml
- Added Unbreakable flag trough NMS to bows, swords and axes.
- Optimized a bit the main command
- Added the Play Again command (bw playagain, executable only in spectator, and I believe only through the spectator item) and the languages lines to English and Italian, other languages needs to be updated
- Fixed spaces on the plugin.yml LOL it was triggering me
- Added permissions to /bw join and /bw gui, now the permissions are: bw.join.GROUP or bw.join.random, and bw.gui.GROUP
- Fixed /bw arenagroup list
- Now the parties won't disband if the owner leaves an arena

ALL OF THESE THINGS WENT TESTED WITH 5 PLAYERS ONLINE - Thanks to Xx_ruben2005_xX for the support

Next updates by me: Hotbar manager, Map selector with favourites & challanges.

Commit:
Update:
- Option to enable halloween features while in halloween period through the config.yml
- Added Unbreakable flag trough NMS to bows, swords and axes.
- Optimized a bit the main command
- Added the Play Again command (bw playagain, executable only in spectator, and I believe only through the spectator item) and the languages lines to English and Italian, other languages needs to be updated
- Fixed spaces on the plugin.yml LOL it was triggering me
- Added permissions to /bw join and /bw gui, now the permissions are: bw.join.GROUP or bw.join.random, and bw.gui.GROUP
- Fixed /bw arenagroup list
- Now the parties won't disband if the owner leaves an arena

ALL OF THESE THINGS WENT TESTED WITH 5 PLAYERS ONLINE - Thanks to Xx_ruben2005_xX for the support

Next updates by me: Hotbar manager, Map selector with favourites & challanges.

Commit:
Merge remote-tracking branch 'github/master'

marioded committed:
Merge branch 'andrei1058:master' into master

Commit:
Merge branch 'master' of https://github.com/zMario34/BedWars1058

 Conflicts:
	bedwars-plugin/src/main/java/com/andrei1058/bedwars/arena/Arena.java
